common/buildings/hfe_buildings.txt

```diff
@@ -1,15 +1,24 @@
 #
 
-hfe_building_terraformer = {
+hfe_building_terraformer = { # Temple of Thoth
 	icon = building_great_pyramid
+	base_cap_amount = 1
 	base_buildtime = @fetime
-	can_build = no
 	can_demolish = no
+	can_be_ruined = no
+	can_be_disabled = yes
 	category = research
 	potential = {
 		exists = owner
 		owner = { is_gestalt = no }
+		OR = {
+			owner = { has_country_flag = hfe_egypt }
```

Comment:
It is. It cannot exist anywhere except the Home of Light. That line of code allows its rebuilding if, somehow, it is destroyed. It cannot go elsewhere.

Reply:
You dont need owner check then.
Now it mean or ju ahet or planet and it mean ju ahet can build it on every planet

Reply:
That Potential check means that the Ju Ahets (or players) can _rebuild_ the _EXISTING_ Temple of Thoth. They cannot erect a new one elsewhere.

Reply:
No, this is or
owner have flag or planet have flag

Reply:
As I said before, this ensures that if the Temple of Thoth is destroyed _on the Ju Ahet's Home of Light capital_, it can be rebuilt on that same spot.

Reply:
Read you trigger again
It will make ju ahet build this building anywhere
Open game, if you do not belive me

Reply:
Okay, done